CanLII.js

```diff
@@ -76,17 +69,17 @@ function scrape(doc, url) {
 		newItem.firstPage = reporterDetails[3];
 	}
 	
-	newItem.court = text('#breadcrumbs span', 2);
+	newItem.court = text('#breadcrumbs *[itemprop="name"] a', 2);
```

Comment:
No need for the `a`, though — it doesn't add anything and just makes it slightly more brittle (e.g., if they change it to use `a` with `itemprop="name"` directly without `span`).

Reply:
(Though I guess in that case the itemprop value would be the href, so they wouldn't do that, exactly, but still, no need for the extra `a`.)